client/components/FormattingBar/controlComponents/ControlsLink.tsx

```diff
@@ -61,6 +107,198 @@ const ControlsLink = (props: Props) => {
 		}
 	};
 
+	const checkedOpenInNewTab = activeLink.attrs.target === '_blank';
+
+	const handleLinkAttr = () => {
+		setIsLoadingAttr(true);
```

Comment:
Was this necessary to get the attribute to update in the DOM? If not, we might be able to remove this state.